packages/app-store/_utils/getCalendar.ts

```diff
@@ -1,13 +1,12 @@
-import { Credential } from "@prisma/client";
-
 import logger from "@calcom/lib/logger";
 import type { Calendar } from "@calcom/types/Calendar";
+import { CredentialPayload } from "@calcom/types/Credential";
 
 import appStore from "..";
 
 const log = logger.getChildLogger({ prefix: ["CalendarManager"] });
 
-export const getCalendar = (credential: Credential | null): Calendar | null => {
+export const getCalendar = (credential: CredentialPayload | null): Calendar | null => {
```

Comment:
Most of the changes includes changing the type that was being used for Credential.